emsdk.py

```diff
@@ -2614,15 +2611,9 @@ def construct_env(tools_to_activate):
       value = to_native_path(tool.expand_vars(value))
       env_vars += [(key, value)]
 
-  # Don't set env vars which are already set to the correct value.
```

Comment:
i don't know why this existed. are you sure it's not needed?

Reply:
For PATH manipulation I think it was useful so that repeatably running this script doesn't grow your PATH indefinitely. 

For setting env vars, and I cant' see any reason not to just set them.

It does mean the script won't ever be completely silent... but I think that is fine, people shouldn't generally be running it repeatedly anyway.